package.json

```
@@ -11,7 +11,7 @@
     "preview": "astro preview",
     "clean": "npx -y rimraf ./node_modules ./.astro ./dist",
     "check": "astro check && npm run tsc -b && npm run typo && npm run lint",
-    "lint": "eslint",
+    "lint": "eslint --fix",
```

[IMO]

この NPM Script は CI/CD で使う予定なので, 引数 `--fix` は無いほうが嬉しいかもです.  理由としては, Auto Fix (`--fix`) が CI/CD 環境で適用されてしまうと, 手元のコードとの乖離が発生してデバッグが困難になってしまうからです.  (Auto Fix が実害を及ぼす編集をする可能性は低いですが……)
もし, 引数 `--fix` を削除しても開発中にまとめて Auto Fix したいときは, `npm run lint --fix` でも引数はきちんと渡されます.  ぜひご検討ください.

```suggestion
    "lint": "eslint",
```